llvm/test/CodeGen/X86/avx512-gather-scatter-intrin-deprecated.ll

```diff
@@ -268,30 +268,6 @@ define void @gather_qps(<8 x i64> %ind, <8 x float> %src, ptr %base, ptr %stbuf)
   ret void
 }
 
-declare  void @llvm.x86.avx512.gatherpf.qps.512(i8, <8 x i64>, ptr , i32, i32);
```

Comment:
Surprise to see they were working without a avx512pf feature.

Reply:
yes, this should be a bug before.